generator/vulkan/parse.zig

```diff
@@ -205,6 +205,18 @@ fn parseContainer(allocator: *Allocator, ty: *xml.Element, is_union: bool) !regi
 
     members = allocator.shrink(members, i);
 
+    var maybe_extends: ?[][]const u8 = null;
+    if (ty.getAttribute("structextends")) |extends| {
+        const n_structs = std.mem.count(u8, extends, ",") + 1;
+        maybe_extends = try allocator.alloc([]const u8, n_structs);
+        var struct_extends = std.mem.split(extends, ",");
```

Comment:
Not sure if this is the most efficient way to do this - one could prob write a custom `u8` function that does this more efficiently, but this is simpler and utilizes what's provided by `std.mem`.